jobs/function_test/prepare.sh

```diff
@@ -75,16 +75,8 @@ preparePackages() {
         popd
     done
 
-    cp -r build-deps/RackHD .
-    if [ -d "build-deps/on-build-config" ]; then
-        cp -r build-deps/on-build-config build-config
-    else
-        cp -r on-build-config build-config
-    fi
-    pushd build-config
-    ./build-config
-    popd
 
```

Comment:
I don't think It's a good idea to keep one entry.
If there is a PR of RackHD which depends on a PR of on-build-config, the PR of on-build-config won't be tested totally